etc/.golangci.yaml

```diff
@@ -17,43 +18,50 @@ linters:
     - contextcheck
     - cyclop
     - deadcode
+    - depguard
     - exhaustivestruct
     - exhaustruct
     - forcetypeassert
     - funlen
-    - gocognit
-    - godox
-    - goerr113
     - gochecknoglobals
     - gochecknoinits
+    - gocognit
+    - goconst
```

Comment:
This complained about seeing something a handful of times. I think it liked to target test files.